src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnCredential.java

```diff
@@ -0,0 +1,169 @@
+package it.smartcommunitylab.aac.webauthn.persistence;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.yubico.webauthn.RegisteredCredential;
+import com.yubico.webauthn.data.AuthenticatorTransport;
+import com.yubico.webauthn.data.ByteArray;
+
+@Entity
```

Comment:
add @table annotation with table name webauthn_credentials

Reply:
Fixed in 57fa46680194772f79ad09b469e55c45fe1e071d